compiler/vmdef.nim

```diff
@@ -238,6 +237,7 @@ type
     slots*: ptr UncheckedArray[TFullReg]
     currentException*: PNode
     currentLineInfo*: TLineInfo
+    callbackNode*: PNode
```

Comment:
Why is this required? We have dozens of callbacks already which work fine without this.